makefile

```diff
@@ -15,6 +15,9 @@ GDC ?= gdc
 
 dmd: bin/dfmt
 
+views/VERSION : $(SRC)
+	git describe --tags | cut -c2- | cut -f1-2 -d- > views/VERSION
```

Comment:
This needs a comment, what are you cutting and why? Also, use `> $@` instead of `> views/VERSION`, so if for some reason the file is renamed you don't need to update the script too.

Reply:
Also, this probably needs to get built unconditionally. If you change a README file and commit for example, the git hash /tag you are building will change, but the code will say you built another version. While the binary would be the same, it seems misleading.

You can unconditionally generate this file every time `make` runs via this trick:

```make
__dummy_version := $(shell git describe --tags | cut -c2- | cut -f1-2 -d- > views/VERSION)
```